core/services/chainlink/application.go

```diff
@@ -292,6 +293,12 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 	}
 	app.HeadTracker = services.NewHeadTracker(store, headTrackables)
 
+	head, err := app.HeadTracker.HighestSeenHeadFromDB()
```

Comment:
hmm do you have in mind some approach? HeadTracker is created as a last service (and with LB as dependency), so i can't pass it to LB directly

Reply:
I do think we can change this, but let's save it for a future PR.

Reply:
@RyanRHall the problem with doing that is it implicitly relies on HeadTracker being started afterwards.

If someone were to move it around in future this would not be obvious and could easily be broken resulting in a rare and difficult to debug race condition.

I think its right to leave it here, and we may even want to add a comment explaining why ordering is important.

Reply:
done 👍🏼